src/Illuminate/Database/Schema/Builder.php

```diff
@@ -94,6 +94,28 @@ public static function morphUsingUuids()
         return static::defaultMorphKeyType('uuid');
     }
 
+    /**
+     * Create a database in the schema if the database not exists.
+     *
+     * @param  string  $name
+     * @return bool
+     */
+    public function createDatabaseIfNotExists($name)
+    {
+        throw new LogicException('This database driver does not support create databases.');
```

Comment:
The message should be changed to `This database driver does not support creating databases.`

Reply:
Done. Thank you.